RUFAS/output_manager.py

```diff
@@ -149,6 +159,16 @@ def __init__(self) -> None:
             )
             self.time = None
             self._variables_usage_counter: Counter[str] = collections.Counter()
+            self.is_end_to_end_testing_run: bool = False
+            self.is_first_post_processing: bool = True
+
+    @property
+    def _filter_prefixes(self) -> dict[str, str]:
+        """Returns the appropriate set of acceptable filter prefixes."""
+        if not self.is_end_to_end_testing_run:
+            return self.__supported_filter_types_prefixes
+        else:
+            return self.__end_to_end_testing_filter_prefixes
```

Comment:
```suggestion
        if self.is_end_to_end_testing_run:
            return self.__end_to_end_testing_filter_prefixes
        else:
            return self.__supported_filter_types_prefixes
```
avoid unnecessary negation